071-GitHubCopilot/Coach/Solution-05.md

```diff
@@ -0,0 +1,16 @@
+# Challenge 05 - Debugging and Code Optimization with Copilot - Coach's Guide 
+
+[< Previous Solution](./Solution-04.md) - **[Home](./README.md)**
+
+## Notes & Guidance
+
+- Students can grab a file of their choice from the juice-shop repo.  For example you can choose the order.ts file located under routes folder.
```

Comment:
should you include links to this repo & file?

Reply:
Good idea, I have added them